android_mobile/src/main/java/com/alexstyl/specialdates/ReroutingActivity.kt

```diff
@@ -4,36 +4,63 @@ import android.app.Activity
 import android.app.NotificationChannel
 import android.app.NotificationManager
 import android.app.PendingIntent
+import android.content.ComponentName
 import android.content.Context
 import android.content.Intent
 import android.graphics.Color
 import android.os.Bundle
 import android.support.v4.app.NotificationCompat
 import com.alexstyl.android.Version
 import com.alexstyl.specialdates.home.HomeActivity
-import java.time.Clock
+import javax.inject.Inject
 
 class ReroutingActivity : Activity() {
 
+
+    lateinit var errorTracker: CrashAndErrorTracker
+        @Inject set
```

Comment:
do you need this line? as far as I know if you annotate the var with `@Inject` it works

Reply:
It might have been the Java -> kotlin converter that showed me that that's how you set the annotations when I started using dagger with kotlin. I removed it and annotated the var with `@Inject` and it works wonders. Cheers 👌